recording_rosbag: False
mission_mode: "manual" # 'manual', 'random', or 'predefined'
